.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "frontend/libretro-common"]
+	path = frontend/libretro-common
```

Comment:
Any reason why this is in `frontend`? When I've seen libretro-common in cores, they've just been in the root with `/libretro-common`.